RELEASE_NOTES.md

```diff
@@ -11,3 +11,7 @@
 ### v0.0.1
 
 - Initial release
+
+### Pending
+
+- Added `allowUnpersistedOperation` option, allowing arbitrary operations to be issued under controlled circumstances
```

Comment:
Please move this above the `### v0.0.1` entry; thanks!

Reply:
Did you mean above `### v0.0.3` ?